packages/MarkdownEditor-Tests.package/MarkdownBlockTextStylerTest.class/instance/setUp.st

```diff
@@ -0,0 +1,4 @@
+running
+setUp
+
+	blockParser := MarkdownBlockTextStyler new
```

Comment:
Why do we still have a blockParser here? I thought they should all be replaced by blockTextStyler.

Reply:
Haha you literally approved and commented seconds after I pushed the changes. This is changed there.